src/resources/UsageAnalytics/Read/Snowflake/SnowflakeInterfaces.ts

```diff
@@ -22,7 +22,13 @@ export interface SnowflakeUserModel {
 }
 
 export interface ReactivateUserParams {
-    daysToExpiry: number;
+    daysToExpiry?: number;
+}
+
+export interface ReactivateUserModel {
+    name: string;
+    email: string;
+    daysToExpiry?: number;
```

Comment:
I think technically the service reports this value as `null`, so the type should be
```suggestion
    daysToExpiry?: number | null;
```

Same could be set for the params by the way, passing `null` instead of omitting the parameter is valid too, I think. (It's the "same" to Jackson, both deserialize to the field being `null` on the Java object).